src/app/metadata/features/dataset-details/dataset-details.component.ts

```diff
@@ -87,10 +85,21 @@ export class DatasetDetailsComponent implements OnInit, AfterViewInit {
   #dins = inject(DatasetInformationService);
 
   #datasetDetails = this.#metadata.datasetDetails;
-  error = this.#datasetDetails.error;
+  #error = this.#datasetDetails.error;
   datasetDetails = this.#datasetDetails.value;
   datasetInformation = this.#dins.datasetInformation.value;
 
+  errorMessage = computed(() => {
+    switch (this.#error()) {
```

Comment:
Shouldn't this be `#error().status?`.

Also, I would use a more specific name like `#datasetDetailsError` (there could also be an error for the dins).

Reply:
You're right! I will fix it